gravitee-apim-console-webui/src/management/api/api-navigation/api-confirm-deployment-dialog/api-confirm-deployment-dialog.component.html

```diff
@@ -32,5 +32,5 @@ <h2 matDialogTitle>Deploy your API</h2>
 </mat-dialog-content>
 <mat-dialog-actions class="actions" align="end">
   <button mat-flat-button [mat-dialog-close]="undefined">Cancel</button>
-  <button color="primary" mat-raised-button aria-label="Deploy the API" (click)="onDeploy()">Deploy</button>
+  <button data-testid="confirmApiDeploy" color="primary" mat-raised-button aria-label="Deploy the API" (click)="onDeploy()">Deploy</button>
```

Comment:
to be consistent with the other testid: 
```suggestion
  <button data-testid="confirm_api_deploy" color="primary" mat-raised-button aria-label="Deploy the API" (click)="onDeploy()">Deploy</button>
```